clusterloader2/testing/load/config.yaml

```diff
@@ -345,6 +354,7 @@ steps:
         templateFillMap:
           ReplicasMin: {{$MEDIUM_GROUP_SIZE}}
           ReplicasMax: {{$MEDIUM_GROUP_SIZE}}
+  {{if not $IS_SMALL_CLUSTER}}
```

Comment:
Instead of havign this if block, we could introduce a variable`$bigJobsPerNamespace` and make sure it will be set to 0 if #Nodes < 100